renovate.json5

```diff
@@ -1,10 +1,51 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
-    "config:base",
+    "config:base"
   ],
-  ignorePaths: [
-      "src/jvmTest/resources/",
+  enabled: true,
+  enabledManagers: [
+    "gradle",
+    "gradle-wrapper",
+    "github-actions",
+  ],
+  // Will auto-merge directly, without a PR, if tests pass - else, makes a PR.
+  // Must add Renovate to 'Allow specified actors to bypass required pull requests'
+  // in branch protection rule
+  automergeType: "branch",
+  platformAutomerge: true,
+  ignoreTests: false,
+  packageRules: [
+    {
+      description: "auto-merge all but major releases",
+      matchUpdateTypes: [
+        "minor",
+        "patch",
+        "pin",
+        "digest",
+      ],
+      automerge: true,
+    }
   ],
-  automerge: true,
+  timezone: "Etc/UTC",
+  schedule: [
+    // loosely limit to Europe daytime, so we don't get pinged in the middle of the night
+    "after 10am", "before 5pm",
+  ],
+  stabilityDays: 14,
```

Comment:
The name of this setting is different now: https://docs.renovatebot.com/configuration-options/#minimumreleaseage

Reply:
BTW, I don't fully understand how it will work. Let's say that v1.2.3  appears on January 1st. Renovate waits 14 days with merging the version bump. On January 7th, v1.2.4 appears. What will Renovate do? Will it even merge v1.2.3, or will it start another waiting period of 14 days after v1.2.4 appears?

Reply:
I don't fully understand either :) 

I think (because Renovate re-uses PRs) any version change will start another waiting period. Which I think is fine - if something has a lot of small releases then 1. it would bug me having loads of notifications 2. it indicates that the library isn't stable, and I'd rather move slowly.

Reply:
Ah just reading the docs you linked, the delay is per-version, not per-library.

> Note: Renovate will wait for the set duration to pass for each separate version. Renovate does not wait until the package has seen no releases for x time-duration